bubuku/broker.py

```diff
@@ -71,6 +71,9 @@ def start_kafka_process(self, zookeeper_address):
         :param zookeeper_address: Address to use for kafka
         :raise LeaderElectionInProgress: raised when broker can not be started because leader election is in progress
         """
+        if not zookeeper_address:
+            raise Exception('No connection to zookeeper')
```

Comment:
According to the way how this string is generated, this if doesn't make any sense
```
','.join(['{}:{}'.format(h, port) for h in hosts]) + self.prefix
```
If there is prefix, than it will return something in any case.